test/test_markdown.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import warnings
+import json
+import pytest
+from jsonschema import validate
+import commonmark
+
+
+parser = commonmark.Parser()
+
+
+with open('element-schema.json') as fp:
+    SCHEMA = json.load(fp)
+
+
+def test_markdown(markdown_file):
+    if markdown_file.samefile('docs/migration.md'):
+        pytest.skip('migrate.md intentionally contains invalid API Elements')
+
+    with open(markdown_file) as fp:
+        ast = parser.parse(fp.read())
+
+    for (node, _) in ast.walker():
```

Comment:
are the parentheses necessary? 🤔

Reply:
Apparently not (in Swift they are though 😬).